merge-v1/build/schema.graphql

```diff
@@ -1,44 +1,118 @@
-"Can be used for marketplace construction or even just a specific visualization front-end"
-type Hodler @entity {
+"""
+Color of NFT from least rare to rarest
+"""
+enum Color {
+  WHITE #tier/class - 1
+  YELLOW #tier/class - 2
+  BLUE #tier/class - 3
+  RED #tier/class - 4
+  BLACK #tier/class - 4
+}
+
+"""
+Class of NFT from least rare to rarest that corresponds to Color
+"""
+enum Class {
+  ONE
+  TWO
+  THREE
+  FOUR
+}
+
+"""
+An account of users holding an NFT within merge. collection.
+"""
+type User @entity {
+  "The holder's address of NFT in collection"
   id: ID!
-  collectionName: String!
-  owner: Bytes!
-  tokenID: BigInt! # uint256
-  massSize: BigInt!
-  rarityPreMerge: BigDecimal! #TODO: percentage of mass NFTs that are this specific mass size in the whole collection. This one is trickier bc we need the instance of total mass at that specific size right before Merging capability was activated. I think this is better in a separate entity. The front end devs can just take MassGroups entities and calculate the rarity.
-  rarityNow: BigDecimal! #TODO: see above comment. Same applies.
-  isBlue: Boolean!
-  isYellow: Boolean!
-  isNormal: Boolean! #TODO: Front end should be able to work with isBlue and isYellow to determine that it isNormal, but I'll put it in and see what DK says.
-  purchasePrice: BigDecimal # original purchase ether price of NFT
-  purchaseDate: BigDecimal #blockNumber converted to human time?
-  tier: BigInt!
-  class: BigInt!
-  mergeCount: BigInt!
-  tokenStandard: String!
+  "Unique tokenID for a respective NFT"
+  tokenID: BigInt!
```

Comment:
How would this work if the user is open sea and they are whitelisted and they can hold multiple?

Reply:
you can do:

```graphql
tokens: [NFTs!]! @derivedFrom(field: owner)
```

Reply:
Great point. I have implemented 

```graphql
  "Details about NFTs being held by this User"
  massNFTs: [NFTs!]! @derivedFrom(field: owner) #pak: allows one to query info about NFTs belonging to this user
```